third_party/bazel/cuda_supplement/conflict_symbols.txt

```diff
@@ -0,0 +1,101 @@
+adler32
```

Comment:
How do you get these conflict symbols? Can we generate this file automatically?

Reply:
It's all symbols of zlib. It's hard to generate on the fly since they're build from source too and not straight forward to generate in cuda_supplement's bazel script. Considering zlib is quite stable and we wanna fix tf's whole archive behavior in the end, a static text file is here.

Reply:
I see. Then I think `zlib_symbols.txt` is better than `conflict_symbols.txt`

Reply:
renamed.